crates/test-utils/src/genesis.rs

```diff
@@ -0,0 +1,995 @@
+//! Utilities for working with a `genesis.json` and other chain config structs.
+
+use crate::{from_bytes_to_h256, serde_helpers::deserialize_stringified_u64_opt};
+use alloy_primitives::{Address, Bytes, B256, U256, U64};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+/// This represents the chain configuration, specifying the genesis block, header fields, and hard
+/// fork switch blocks.
+#[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct Genesis {
```

Comment:
these types I wanted to put in its own crate (extracting from reth-primitives)

but for now this is fine